JuntosFinanzas/scivias
# Sync partner and projects with Mercury

- Adds partner and project models
- Listens to partner and project change events from Mercury and updates Scivias DB accordingly

#### How to test
- Ensure that Partner and Project pages can be viewed on the admin dashboard
- Verify that new partner and projects cannot be created via the admin dashboard UI
